src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,70 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+    private static final String COMA = ",";
+    private static final int INDEX_OF_SUPPLY = 0;
+    private static final int INDEX_OF_BUY = 1;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        writeDataToFile(toFileName, writeDataToSting(calculateDate(readFromFile(fromFileName))));
+    }
+
+    private String readFromFile(String fromFileName) {
+        File file = new File(fromFileName);
+        StringBuilder dataString = new StringBuilder();
+        try {
+            BufferedReader reader = new BufferedReader(new FileReader(file));
```

Comment:
if you are using classes that implement an AutoCloseable interface, we should use it with try-with-resources.